app/assets/stylesheets/nucore.scss

```diff
@@ -632,21 +632,33 @@ li:hover {
 /*UCONN Button Changes*/
 
 .btn, .btn-add {
-  transition: ease-in-out 0.2s all;
+  transition: ease 0.2s all;
   border: solid $useColorDark 1px;
   outline: none;
-  background: transparent;
+  background: none;
+  background-color: transparent;
   border-radius: 8px;
   box-shadow: none;
   color: $useColorDark;
   text-shadow: none;
 }
 
+.btn-primary {
+  font-weight: bold;
+  border-width: 1.5px;
+}
+
 .btn:hover, .btn-add:hover {
+  transition: ease 0.2s all;
```

Comment:
Do we need this? The `.btn` style feels like it should cover it.

Reply:
Typically I wouldn't, but bootstrap is applying a hover transition so I had to override that.

Reply:
Gotcha. If you can clean up the conflicts, I'll merge it!